projects/pgai/pgai/vectorizer/vectorizer.py

```diff
@@ -212,6 +212,18 @@ def migrate_config_to_new_version(cls, data: Any) -> Any:
         logger.warning("Unable to migrate configuration: raw data type is unknown")
         return data  # type: ignore[reportUnknownVariableType]
 
+    def ensure_features(self, features: Features):
```

Comment:
My idea is to have this method in the Vectorizer rather than adding the logic to the Worker directly, as I believe it is really responsibility of the first.

However, I could revert this and move the logic to the worker here https://github.com/timescale/pgai/pull/740/files#diff-14e6985b21bba858a953181a5497b939abb251f98b082774b506b62c5e410b60R127